### PR TITLE
add radial plane

### LINKED ISSUE #111: Add "radial topology on the plane"
- Willard 3.A.4
- This is NOT regular

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #111 but we need to add MR numbers (or other bib info) for a couple references. (I'm off campus today.)

### CODE REVIEW

**Comment by StevenClontz** [comment]
This is another example of a k-space which is neither first-countable nor locally compact: https://topology.pi-base.org/spaces?q=Generated%20by%20compact%20Hausdorff%20subspaces%2B~First%20Countable%2B~Locally%20Compact

**Comment by ccaruvana** [comment]
Had to come up to campus to do a couple of things and decided to check in on some of these MRs.

Klee 1955 Some top... MR0069388
Klee 1951 Convex... MR0044014
There is also a part 2 to the convex sets in linear spaces paper in the same year: MR0047251

File: spaces/S000175/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - mr: MR0069388
+    name: Some topological properites on convex sets (Klee 1955)
+  - mr: MR0044014
+    name: Convex sets in linear spaces (Klee 1951)
+  - doi: 10.2307/2315929
+    name: Problem 5468 of The American Mathematical Monthly
+---
+
+Sets $U$ are open provided for every point $x$ and line passing through $x$,
+an open subinterval of the line is a subset of $U$.
```
Comment (prabau) [inline spaces/S000175/README.md:19]: Mention that $x$ is in $U$ and the open subinterval contains $x$.  Something like "... for every point $x\in U$ and line passing through $x$, the point $x$ is contained in an open subinterval of the line that is a subset of $U$" for example.

**Comment by prabau** [comment]
I was trying to follow the proof from the Monthly that the space is not regular.  In the next to last paragraph, where is says "If $U$ is a $T$-neighborhood of $\alpha$ and $\alpha\in D$", I think it was meant $\alpha\notin D$.  If that's the case, it may be worth mentioning this typo here.

File: spaces/S000175/properties/P000079.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000175
+property: P000079
+value: true
+refs:
+  - doi: 10.2307/2315929
+    name: Problem 5468 of The American Mathematical Monthly
+---
+
+Follows as the space is the quotient of first-countable spaces (Euclidean lines).
```
Comment (prabau) [inline spaces/S000175/properties/P000079.md:10]: Usually a quotient space is with respect to a single quotient map.  We know what we mean here, but do you think we should say something like quotient of a topological sum of ... ?

**Comment by StevenClontz** [comment]
@prabau can you directly make those two edits? They sound good to me.

**Review by prabau — APPROVED** [review]

**Pull request merged**